<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationWdgtCardAppearance</class>
 <widget class="QWidget" name="ConfigurationWdgtCardAppearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Card type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="card_types_widget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontal_layout_bg_colour">
       <item>
        <widget class="QLabel" name="bg_label">
         <property name="text">
          <string>Background:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="background_button">
         <property name="text">
          <string>Set colour</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="align_label">
         <property name="text">
          <string>Alignment:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alignment">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Center</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_non_latin">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="non_latin_layout">
       <item>
        <widget class="QLabel" name="label_non_latin_1">
         <property name="text">
          <string>Increase size of non-latin characters by</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="non_latin_font_size_increase"/>
       </item>
       <item>
        <widget class="QLabel" name="label_non_latin_2">
         <property name="text">
          <string>points.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_non_latin">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_non_latin_3">
       <property name="text">
        <string>(This setting will be overridden in card types cloned from 'Vocabulary'.)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="preview_button">
         <property name="text">
          <string>&amp;Preview</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>preview_button</sender>
   <signal>clicked()</signal>
   <receiver>ConfigurationWdgtCardAppearance</receiver>
   <slot>preview</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>card_types_widget</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ConfigurationWdgtCardAppearance</receiver>
   <slot>card_type_changed</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>background_button</sender>
   <signal>clicked()</signal>
   <receiver>ConfigurationWdgtCardAppearance</receiver>
   <slot>update_background_colour</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alignment</sender>
   <signal>activated(int)</signal>
   <receiver>ConfigurationWdgtCardAppearance</receiver>
   <slot>update_alignment</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
